newIDE/app/src/EventsSheet/ParameterFields/ObjectField.js

```diff
@@ -3,94 +3,90 @@ import * as React from 'react';
 import classNames from 'classnames';
 import { type ParameterInlineRendererProps } from './ParameterInlineRenderer.flow';
 import ObjectSelector from '../../ObjectsList/ObjectSelector';
-import { type ParameterFieldProps } from './ParameterFieldCommons';
+import {
+  type ParameterFieldProps,
+  type ParameterFieldInterface,
+  type FieldFocusFunction,
+} from './ParameterFieldCommons';
 import { Trans } from '@lingui/macro';
 import { nameAndIconContainer } from '../EventsTree/ClassNames';
 import InAppTutorialContext from '../../InAppTutorial/InAppTutorialContext';
 
-export default class ObjectField extends React.Component<
-  ParameterFieldProps,
-  {||}
-> {
-  static contextType = InAppTutorialContext;
+export default React.forwardRef<ParameterFieldProps, ParameterFieldInterface>(
+  function ObjectField(props: ParameterFieldProps, ref) {
+    const field = React.useRef<?ObjectSelector>(null);
+    const focus: FieldFocusFunction = options => {
+      // Prevent focus of field if an in-app tutorial is running because
+      // the popper of the tooltip and the popper of the semi controlled
+      // autocomplete's dropdown are conflicting.
+      if (field.current && !currentlyRunningInAppTutorial)
```

Comment:
should this be done for **every** field when the in app tutorial is running?

Reply:
Maybe: the object field has a particular way of displaying options (in a popper) that maybe other fields have but we didn't encounter the issue for another field.

Reply:
oh right, so only this one is painful then